### PR TITLE
bitwarden-cli: Initial inclusion

### PR DESCRIPTION
**Pull request by silkeh**
Resolves https://github.com/getsolus/packages/issues/324

## Test Plan

- Logged in to my BitWarden account
- Searched for a password

### CODE REVIEW

File: bitwarden-cli/package.yml [hunk 1]
```diff
@@ -0,0 +1,29 @@
+name       : bitwarden-cli
+version    : 2023.8.2
+release    : 1
+source     :
+    - https://github.com/bitwarden/clients/archive/refs/tags/cli-v2023.8.2.tar.gz : bf17802d24fe6b85ab74dce5bb5060ccfb5187d70f08a8473f65194dd3c21303
+license    : GPL-3.0-only
+component  : security
```
Comment (davidjharder) [inline bitwarden-cli/package.yml:7]: needs `homepage:  `

File: bitwarden-cli/package.yml [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+    cd apps/cli
+
+    npm run dist:lin
+
+    node ./build/bw.js completion --shell zsh > _bw
+install    : |
+    cd apps/cli
+
+    install -Dm00755 dist/linux/bw $installdir/usr/bin/bw
+    install -Dm00644 _bw $installdir/usr/share/zsh/site-functions/_bw
```
Comment (davidjharder) [inline bitwarden-cli/package.yml:30]: Why only zsh completions?

File: bitwarden-cli/package.yml (hunk 1 above)
Comment (silkeh) [reply bitwarden-cli/package.yml:8]: Well, I clearly followed the instructions to the letter :disappointed:

File: bitwarden-cli/package.yml (hunk 2 above)
Comment (silkeh) [reply bitwarden-cli/package.yml:30]: It's the only completion provided:

```sh
Usage: bw completion [options]

Generate shell completions.

Options:
  --shell <shell>  Shell to generate completions for.
  -h, --help       display help for command

  Notes:

    Valid shells are `zsh`.

  Examples:

    bw completion --shell zsh

```

Comment (davidjharder) [reply bitwarden-cli/package.yml:30]: ¯\_(ツ)_/¯

**Review by davidjharder — DISMISSED** [review]
LGTM

**Comment by ReillyBrogan** [comment]
Maintaining this package @silkeh? I hope you enjoy changelogs where the only text is "bug fixes"

**Comment by silkeh** [comment]
@ReillyBrogan: well, somebody has to do it :disappointed_relieved:

**Review by ReillyBrogan — APPROVED** [review]
LGTM

**Pull request merged**